src/zodbbrowser/browser.py

```diff
@@ -68,7 +72,25 @@ def __ne__(self, other):
 
 class VeryCarefulView(BrowserView):
 
+    version = __version__
+    homepage = __homepage__
     made_changes = False
+    state = None
+
+    @Lazy
+    def references(self):
+        return queryUtility(IReferencesDatabase)
+
+    @Lazy
+    def supportsVersions(self):
+        db = queryUtility(IDatabase, name='<target>')
+        if db is None:
+            return True         # Assume we can, this only happens in tests.
```

Comment:
"assume we can" doesn't seem grammatically correct.

I don't think there's any ZODB version still in existence that supports [versions](http://zodborg.readthedocs.org/en/latest/documentation/guide/transactions.html#versions).